src/components/jacobian_component/jacobian.sh

```diff
@@ -47,125 +47,128 @@ setup_jacobian() {
+            fi
+        fi
+
+        # Update settings in HISTORY.rc
+        # Only save out hourly pressure fields to daily files for base run
+        if [ $x -eq 0 ]; then
+            if "$HourlyCH4"; then
+                sed -i -e 's/'\''Restart/#'\''Restart/g' \
+                    -e 's/#'\''LevelEdgeDiags/'\''LevelEdgeDiags/g' \
+                    -e 's/LevelEdgeDiags.frequency:   00000100 000000/LevelEdgeDiags.frequency:   00000000 010000/g' \
+                    -e 's/LevelEdgeDiags.duration:    00000100 000000/LevelEdgeDiags.duration:    00000001 000000/g' \
+                    -e 's/LevelEdgeDiags.mode:        '\''time-averaged/LevelEdgeDiags.mode:        '\''instantaneous/g' HISTORY.rc
             fi
-	fi
-
-	# Update settings in HISTORY.rc
-	# Only save out hourly pressure fields to daily files for base run
-	if [ $x -eq 0 ]; then
-	    if "$HourlyCH4"; then
-		sed -i -e 's/'\''Restart/#'\''Restart/g' \
-                       -e 's/#'\''LevelEdgeDiags/'\''LevelEdgeDiags/g' \
-                       -e 's/LevelEdgeDiags.frequency:   00000100 000000/LevelEdgeDiags.frequency:   00000000 010000/g' \
-                       -e 's/LevelEdgeDiags.duration:    00000100 000000/LevelEdgeDiags.duration:    00000001 000000/g' \
-                       -e 's/LevelEdgeDiags.mode:        '\''time-averaged/LevelEdgeDiags.mode:        '\''instantaneous/g' HISTORY.rc
-	    fi
-	    # For all other runs, just disable Restarts
-	else
+            # For all other runs, just disable Restarts
+        else
             if "$HourlyCH4"; then
-		sed -i -e 's/'\''Restart/#'\''Restart/g' HISTORY.rc
+                sed -i -e 's/'\''Restart/#'\''Restart/g' HISTORY.rc
             fi
-	fi
+        fi
 
-	# Create run script from template
-	sed -e "s:namename:${name}:g" ch4_run.template > ${name}.run
-	rm -f ch4_run.template
-	chmod 755 ${name}.run
+        # Create run script from template
+        sed -e "s:namename:${name}:g" ch4_run.template >${name}.run
+        rm -f ch4_run.template
+        chmod 755 ${name}.run
 
-	### Turn on observation operators if requested, only for base run
-	if [ $x -eq 0 ]; then
-    	    activate_observations
-	fi
+        ### Turn on observation operators if requested, only for base run
+        if [ $x -eq 0 ]; then
+            activate_observations
+        fi
 
-	### Perform dry run if requested, only for base run
-	if [ $x -eq 0 ]; then
+        ### Perform dry run if requested, only for base run
+        if [ $x -eq 0 ]; then
             if "$ProductionDryRun"; then
-		printf "\nExecuting dry-run for production runs...\n"
-		./gcclassic --dryrun &> log.dryrun
-		./download_data.py log.dryrun aws
+                printf "\nExecuting dry-run for production runs...\n"
+                ./gcclassic --dryrun &>log.dryrun
+                # prevent restart file from getting downloaded since
+                # we don't want to overwrite the one we link to above
+                sed -i '/GEOSChem.Restart/d' log.dryrun
```

Comment:
@msulprizio this is the only actual change that is not just a change in whitespace.